highfidelity/hifi
# accept forced physics results from entity server (e.g., when filtered)

test plan:
- Try the following two filters: [sandbox-zone-filter](https://s3-us-west-1.amazonaws.com/hifi-content/zfox/sandbox-filter.js) and [maxVelocity-filter](https://s3-us-west-1.amazonaws.com/hifi-content/zfox/maxVelocity-filter.js). Use the test plan from https://github.com/highfidelity/hifi/pull/9558. Those two filters should work at least as well as in #9558.
- Now, try the following two filters, which are slightly modified versions of the above filters: [sandbox-zone-filter-noRandomness](https://s3-us-west-1.amazonaws.com/hifi-content/zfox/sandbox-filter-noRandomness.js) and [maxVelocity-filter-noRandomness](https://s3-us-west-1.amazonaws.com/hifi-content/zfox/maxVelocity-filter-noRandomness.js). Use the test plan from #9558.
Does it still work? (Compare to not using the client from this PR. Without the random, we expect the master client to not be good.)
- Without any filters, makes sure that physics and near/far grab work well, especially when there are multiple people. Especially:
 -- Turn on physics ownership markers, and move a physical object within a pile of them, with multiple people present. Make sure the objects settle as they are supposed to, and don't jump unexpectedly.
 -- Use physical toys such as planky, bow and arrow, ping pong. Make sure it's as good as master.
- Try the following filter (either for a specific region or everywhere).  You should be able to interact with objects physically, but not make any other changes. (Near/far grabbing becomes unsatisfying. It "works" only to the extent that it looks like physics. The near/far gab itself is countermanded.
